src/commands/MooCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Message } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import { SingleMention } from '../mentions/SingleMention';
+
+export default class PingCommand extends PrefixCommand {
+	public readonly aliases = ['moo', 'test'];
+
+	public async run( message: Message, args: string ): Promise<boolean> {
+		if ( args.length ) {
+			return false;
+		}
+
+		try {
+			new SingleMention("MC-772");
+		} catch {
+			return false;
+		}
+
+		try {
+			await message.react( '🐄' );

```

Comment:
Should these reactions be added to the bot's message or to the original command? Currently they're added to the original command.

Also, this code works, but you could also look into replacing this by the `reactToMessage` method from `ReactionsUtil`, which is made for adding multiple emojis at once.

Reply:
Switched both of these over